### PR TITLE
CP-1182 Release route.dart 0.7.0

### PR DESCRIPTION
**Pull request by jayudey-wf**
JIRA: https://jira.webfilings.com/browse/CP-1182

JIRA and PR's included in this release: 

 CP-1183  - refactor to permit HistoryProviders and in memory history + dart_dev  - https://github.com/Workiva/route.dart/pull/1

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/route.dart/compare/0.6.1...CP-1182_release_0.7.0

### CODE REVIEW

**Comment by evanweible-wf** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "url": "https://github.com/angular/route.dart.git"
   },
   "devDependencies": {
-    "karma" : "~0.12.0",
-    "karma-dart" : "~0.2.6",
+    "karma": "~0.12.0",
```
Comment (trentgrover-wf) [inline package.json:10]: note to self: kill all this stuff when setting up travis...

**Comment by trentgrover-wf** [comment]
+1

**Comment by jayudey-wf** [comment]
QA Resource Approval: +1
- [x] Dev +1's only needed for Release ticket

Ready for merge and tag.

**Comment by teresarevious-wf** [comment]
@rmconsole-wf

**Pull request merged**